doc/ref_model/chapters/chapter05.md

```diff
@@ -255,13 +289,20 @@ The configurations specified in here will be used in specifying the actual hardw
 <a name="5.4.1"></a>
 ### 5.4.1 Compute Resources
 
-
+| Reference | Feature | Description | Basic Type | Network Intensive |
+|---------------------|-----------|---------------------------|--------|--------|
+| nfvi.hw.cpu.cfg.001 | Number of CPU (Sockets) | This determines the minimum number of CPU sockets within each host | 2| 2|
+| nfvi.hw.cpu.cfg.002 | Number of Cores per CPU | This determines the number of cores needed per each CPU. | 20 | 20 |
```

Comment:
nfvi.hw.cpu.cfg.002 should be "Minimum Number of Cores per CPU" instead of "Number of Cores per CPU"

Reply:
@pgoyal01 your reviews here are not related to the purpose of the PR. If it OK with you, I will stick to the scope of the PR if possible.